sample-code/spring-app/pom.xml

```diff
@@ -170,6 +170,10 @@
           <groupId>ch.qos.logback</groupId>
           <artifactId>logback-classic</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.yaml</groupId>
+          <artifactId>snakeyaml</artifactId>
+        </exclusion>
```

Comment:
Exclusion is not the recommended way to fix dependency-convergence errors. You can find in SAP Cloud SDK, we use dependencyManagement to override versions for all transitive occurrences
https://github.com/SAP/cloud-sdk-java/blob/main/pom.xml